test/unit.test.js

```diff
@@ -34,21 +34,36 @@ describe('matchBrowserLocale', () => {
     const { matchBrowserLocale } = await import('../src/templates/utils-common')
 
     // Both locales match first browser locale - full locale should win.
-    const appLocales = ['en', 'en-US']
+    const appLocales = ['en', 'en-US', 'en-us']
     const browserLocales = ['en-US', 'en']
 
     expect(matchBrowserLocale(appLocales, browserLocales)).toBe('en-US')
+
+    const appLocales = ['zh-CN', 'zh-cn']
```

Comment:
Please create separate test blocks for tests related to case instead of mixing with existing tests.

Reply:
Got it. Thanks